release_bot/releasebot.py

```diff
@@ -167,24 +167,18 @@ def pr_handler(success):
             msg = f"Version ({latest_gh_str}) is already released and this issue is ignored."
             self.logger.warning(msg)
             return False
-        msg = ""
-        if self.conf.dry_run:
-            msg = (f"I would make a new PR for release of version "
-                   f"{self.new_pr['version']} based on the issue.")
-        else:
-            msg = (f"Making a new PR for release of version "
-                   f"{self.new_pr['version']} based on the issue.")
-        self.logger.info(msg)
+        msg = (f"Making a new PR for release of version "
+               f"{self.new_pr['version']} based on the issue.")
+        if not self.conf.dry_run:
+            self.logger.info(msg)
```

Comment:
but nothing is logged if dry_run, isn't it?

Reply:
Oh, that is being logged in [`github.py`](https://github.com/Aniket-Pradhan/release-bot/blob/dry-run/release_bot/github.py#L431).